test/js/ui/map.test.js

```diff
@@ -930,6 +931,21 @@ test('Map', function(t) {
         t.end();
     });
 
+    t.test('workerCount option', function(t) {
+        var map = createMap({});
+        // TODO: it's not great to check this private member here; better would
+        // be to mock Style and then just check that the correct workerCount
+        // param is being passed.
```

Comment:
We have a policy against checking `// TODO` comments into `master` because they tend to get forgotten. You should either resolve the todo or open a ticket.